frame/mosaic/src/lib.rs

```diff
@@ -714,6 +770,44 @@ pub mod pallet {
 			}
 			Ok(().into())
 		}
+
+		// TODO: Set proper pallet weight
+		#[pallet::weight(1_0000)]
+		pub fn add_remote_amm_id(
+			origin: OriginFor<T>,
+			network_id: NetworkIdOf<T>,
+			amm_id: RemoteAmmIdOf<T>,
+		) -> DispatchResultWithPostInfo {
+			T::ControlOrigin::ensure_origin(origin)?;
+
+			ensure!(
+				!RemoteAmmWhitelist::<T>::contains_key(&network_id, &amm_id),
+				Error::<T>::RemoteAmmIdAlreadyExists,
+			);
+
+			RemoteAmmWhitelist::<T>::insert(network_id, amm_id, ());
+
+			Ok(().into())
+		}
+
+		// TODO: Set proper pallet weight
```

Comment:
Could we get rust doc comments so the scrapper can account for them?

Reply:
done